android/src/main/java/com/pagopa/ioreactnativehttpclient/IoReactNativeHttpClientModule.kt

```
@@ -109,6 +101,8 @@ class IoReactNativeHttpClientModule(reactContext: ReactApplicationContext) :
         val cookie = Cookie(name, value, path = path, domain = domainHost)
         inMemoryCookieStorage.addCookie(domain, cookie)
       } catch (_: Exception) {
+        // TODO: what about handling this scenario?
```

The main point behind this global try-catch is to mimic iOS behaviour: there, there is no exception raised but a silent failure on bad input parameters.

Here, code may throw, either during URL construction or later in `addCookie` (even if latter does not happen, since it fails for the same reason as former). We do not want the application to crash but it also have to resemble the same behaviour on both platforms, in order to ease the usage from the react-native side.

That is why there is a "global" try-catch with an unhandled catch body

---

Okay then. I've removed the `TODO` comment ;)